web-server/v0.2/js/controllerDashboard1.js

```diff
@@ -18,35 +18,63 @@ function clickHandler() {
 
 //Data Handling Functions
 
-/*function readJSON(directory) {
-	var request = new XMLHttpRequest();
-	request.open("GET", directory, false);
-	request.send(null);
-	var jsonObject = JSON.parse(request.responseText);
-	createControllerDivs(jsonObject._source.run.controller, jsonObject._source.run.date, 1);
-}*/
-
 function retrieveJSON() {
 	//Controller and results count handler
-	$.getJSON('http://es-perf44.perf.lab.eng.bos.redhat.com:9280/_search?search_type=count&source={ "aggs": { "run": { "terms": { "field": "controller", "size": 0 } } } }', function (data) {
-		for (i = 0; i <= data.aggregations.run.buckets.length; i++) 
+	$.getJSON('http://es-perf44.perf.lab.eng.bos.redhat.com:9280/pbench.pbench-0.*/_search?search_type=count&source={ "aggs": { "run": { "terms": { "field": "controller", "size": 0 } } } }', function (data) {
```

Comment:
@gurbirkalsi, can you file an issue to remove the hard-coded Elasticsearch hostname reference?